src/tagmanager.cpp

```diff
@@ -65,6 +65,73 @@ int TagManager::tag_add_command(Input input, Output output) {
     return 0;
 }
 
+int TagManager::removeTag(Input input, Output output) {
+    if (input.size() < 2) {
+        return HERBST_NEED_MORE_ARGS;
+    }
+    input.shift();
+    auto tagNameToRemove = input.front();
+    input.shift();
+    auto targetTagName = input.empty() ? get_current_monitor()->tag->name : input.front();
+
+    auto targetTag = find(targetTagName);
+    auto tagToRemove = find(tagNameToRemove);
+
+    if (tagToRemove == nullptr) {
```

Comment:
Here I would prefer `if (!tagToRemove)`.

Reply:
I was under the impression that it's actually not recommended (anymore) to use pointers as logical expressions. I fail to find any evidence for that, so I'll gladly go with your suggestion.